server/routes/termdb.config.ts

```diff
@@ -306,3 +305,11 @@ function getAllowedTermTypes(ds) {
 
 	return [...typeSet]
 }
+
```

Comment:
If `forbiddenRoutes` is returned together with `clientAuthResults` by a new authApi method, then no need for this extra function, since `embedder` is used only to determined forbidden routes within `getSupportedChartTypes()`.